pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -87,12 +94,17 @@ func (m *managerImpl) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAd
 	if len(m.nodeConditions) == 0 {
 		return lifecycle.PodAdmitResult{Admit: true}
 	}
-	notBestEffort := qos.BestEffort != qos.GetPodQOS(attrs.Pod)
-	if notBestEffort {
-		return lifecycle.PodAdmitResult{Admit: true}
+
```

Comment:
i was going to try and structure it as a series of actions, this call was not around when this PR was opened.  will think on the best way to structure it cleanly in a follow-on.